style.css

```diff
@@ -1 +1,180 @@
 /*  Add your styling here */
+* {
+  box-sizing: border-box;
+}
+
+main {
+  background-color: #ffc8dd;
+}
+
+/* bodygrid layout */
+body {
+  display: grid;
+  font-family: "Dancing Script", cursive;
+  grid:
+    "hero"
+    "body"
+    "footer";
+}
+
+header {
+  grid-area: hero;
+}
+
+main {
+  grid-area: body;
+}
+
+footer {
+  grid-area: footer;
+}
+
+/* headersection starts here */
+.nav-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  background-color: #cdb4db;
+  width: auto;
+  height: 300px;
+  /* border: 1px solid black; */
+}
+
+.nav-img {
+  width: 100%;
+  height: auto;
```

Comment:
I think this combination is what is making the image scale in a slightly strange-looking way - it's effectively locking the aspect ratio of the original image. Here's a screenshot of what it looks like on a certain size:

<img width="391" alt="image" src="https://user-images.githubusercontent.com/77307047/185601727-be393eb0-3eb9-48d4-a741-ad9f1763346f.png">

There's no easy or obvious answer for me here - it depends how you want the image to behave, but you could try something like:

```
.nav-img {
  width: 100%;
  height: 100%;
  object-fit: cover;
  object-position: 100% 0;
}
```

`object-fit: cover` would preserve the original aspect ratio, but the image still occupies all the available space.

 `object-position: 100% 0` makes it display the right hand side of your image first - so you see the right hand side as long as possible when scaling down (which I think was the most interesting part of your chosen image)